rspace/src/main/scala/coop/rchain/rspace/ReportingRspace.scala

```diff
@@ -111,25 +112,16 @@ class ReportingRspace[F[_]: Concurrent: ContextShift: Log: Metrics: Span, C, P,
     * Seq(Seq[ReportingEvent](Precharge), Seq[ReportingEvent](userDeploy), Seq[ReportingEvent](Refund))
     * It would be seperated by the softcheckpoint creation.
     */
-  val report: SyncVar[Seq[Seq[ReportingEvent]]] = create[Seq[Seq[ReportingEvent]]](Seq.empty)
-  val softReport: SyncVar[Seq[ReportingEvent]]  = create[Seq[ReportingEvent]](Seq.empty)
+  val report: Ref[F, Seq[Seq[ReportingEvent]]] = Ref.unsafe(Seq.empty[Seq[ReportingEvent]])
+  val softReport: Ref[F, Seq[ReportingEvent]]  = Ref.unsafe(Seq.empty[ReportingEvent])
```

Comment:
This PR now contains both changes combined. One of the reasons to have these changes separately is for you to see how depended PR can be managed.

There are two ways:
1. Both PR implement their own changes completely separate (can introduce some conflicts)
2. One PR is built on commits from the other.

Now that you already have changes from the first PR, you can rebase this one on top of it. So commits in this PR will have all commits from the first one with additional changes from this PR.
When the first PR is merged to dev branch, you rebase this PR on top of the dev branch which will leave only commits from this PR.

Reply:
@SivkovAV Oh, sorry. I made a mistake with the PR changing RSpace and RSpaceOps files.

You have correctly separated these two PRs as I specified in the linked issue.